main.go

```diff
@@ -96,7 +100,7 @@ func main() {
 			ForwardedProto: forwardedProto,
 		},
 	}
-	hydraClientMaker := getHydraClientMaker(defaultSpec)
+	hydraClientMaker := getHydraClientMaker(defaultSpec, tlsTrustStore, insecureSkipVerify)
```

Comment:
I would prefer to have some parsing here, verify if the given path is a correct path and fail if not